app/views/content/funding-your-training.md

```diff
@@ -138,12 +138,16 @@ You may be able to get extra financial support if you have children or other car
 * [Adult Dependants’ Grant](https://www.gov.uk/adult-dependants-grant)
 
 
-## Extra financial support for students with disabilities
+## If you're disabled
+
+You may be able to [get support if you’re a student and you’re disabled, have a learning difficulty or health problem](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim).
```

Comment:
```suggestion
You may be able to [get support if you're a student with a disability, learning difficulty or health problems](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim).
```

Reply:
Either feels okay to me - your call @EmmaFrith

Reply:
I've been using 'if you're disabled' elsewhere on the site in line with the equality and human rights commission style

Reply:
Ah my suggestion was to make the three elements of the list 'equal' rather than rewording them.

It's currently `(disabled and (learning difficulty or health problem)` but I'm guessing it should be `disabled or learning difficulty or health problem`

Reply:
Actually, scratch that. Original makes sense.